examples/integrations/yolo/import_yolo_annotations.ipynb

```diff
@@ -0,0 +1,312 @@
+{
```

Comment:
<p>We have created our labels and can import them to our project. For more information on importing annotations, see [import image annotations](<a href="https://docs.labelbox.com/reference/import-image-annotations)" target="_blank">https://docs.labelbox.com/reference/import-image-annotations)</a>.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Labelbox/labelbox-python/pull/1673/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>

Reply:
Done